.vscode/launch.json

```diff
@@ -0,0 +1,75 @@
+{
```

Comment:
Are these commited on purpose? If not, you could add a new rule to `.gitignore`.